datahub-web-react/src/app/shared/admin/HeaderLinks.tsx

```diff
@@ -107,6 +110,20 @@ export function HeaderLinks(props: Props) {
                     </Link>
                 </LinkWrapper>
             )}
+            {showManagePosts && (
```

Comment:
another thing we're going to need is a new permission for users to be able to manage these posts. So maybe a simple platform privilege `MANAGE_POSTS` or something. An example would be like `MANAGE_GLOSSARIES` privilege. This way datahub users can gate who is actually updating their home page with these posts - so that means some more java work as well